enclave-runner/src/usercalls/mod.rs

```diff
@@ -602,50 +904,48 @@ impl UsercallExtension for UsercallExtensionDefault{
 
 #[allow(unused_variables)]
 impl RunningTcs {
-    fn entry(
+
+    fn entry_async(
         enclave: Arc<EnclaveState>,
         tcs: StoppedTcs,
-        mode: EnclaveEntry,
-    ) -> StdResult<(StoppedTcs, (u64, u64)), EnclaveAbort<EnclavePanic>> {
+        io_send_queue: &mpsc::Sender<(ThreadResult<ErasedTcs>, RunningTcs, EnclaveEntry, RefCell<[u8;1024]>)>,
```

Comment:
Create a type alias or `struct` for the items in this queue so you don't have to repeat the type everywhere.